### PR TITLE
Fixed LP Bugs: Bug #1190194 and Bug #1190983

### PR DESCRIPTION
**Pull request by hrvojem**

### CODE REVIEW

File: doc/source/flexibility/improved_memory_engine.rst [hunk 1]
```diff
@@ -61,6 +61,11 @@ Minimum Block Size
 
 The block size has to be big enough to store all fixed-length information in the first block. If not, the ``CREATE TABLE`` or ``ALTER TABLE`` statements will fail (see below).
 
+Limitations
+===========
+
+|MyISAM| tables are still used for temporary tables where the ``MEMORY`` tables with dynamic row format could be used instead: for schemas containing large ``VARCHAR``s, ``BLOB``, and ``TEXT`` columns.
```
Comment (laurynas-biveinis) [inline doc/source/flexibility/improved_memory_engine.rst]: MyISAM tables are still used for query optimiser internal temporary tables where the MEMORY tables could be used now instead: for temporary tables containing large ...

File: doc/source/management/innodb_expanded_fast_index_creation.rst [hunk 2]
```diff
@@ -4,16 +4,10 @@
 Expanded Fast Index Creation
 ============================
 
-Percona has implemented several changes related to |MySQL|'s fast index creation feature. This feature expands the ``ALTER TABLE`` command by adding a new clause that provides online index renaming capability, that is renaming indexes without rebuilding the whole table.
-
-Enabling Expanded Fast Index Creation
-=====================================
-
-Fast index creation was implemented in |MySQL| as a way to speed up the process of adding or dropping indexes on tables with many rows. However, cases have been found in which fast index creation creates an inconsistency between |MySQL| and |InnoDB| data dictionaries.
+Percona has implemented several changes related to |MySQL|'s fast index creation feature. Fast index creation was implemented in |MySQL| as a way to speed up the process of adding or dropping indexes on tables with many rows. However, cases have been found in which fast index creation creates an inconsistency between |MySQL| and |InnoDB| data dictionaries.
```
Comment (laurynas-biveinis) [inline doc/source/management/innodb_expanded_fast_index_creation.rst]: Hmm what cases? If nobody knows, maybe we can remove this clause and wait for bug reports?

**Comment by hrvojem** [comment]
Comments addressed

**Comment by laurynas-biveinis** [comment]
LGTM

**Pull request merged**